src/renderer/components/dock/logs/download-logs-dropdown.tsx

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Everything here could be done with reusable dropdown / activatable component. Read above